src/ray/raylet/worker_pool.cc

```diff
@@ -118,6 +169,23 @@ void WorkerPool::StartWorkerProcess(const Language &language) {
   for (auto const &token : state.worker_command) {
     worker_command_args.push_back(token.c_str());
   }
+
+  string pydevd_port = find_pydevd_port();
+  if (pydevd_port != "") {
+      worker_command_args.insert(worker_command_args.begin()+1, "/home/pimpke/apps/pycharm-2018.1.4/helpers/pydev/pydevd.py");
```

Comment:
Hardcoded line?

Reply:
Moved this to python, in services.start_raylet.